src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java

```diff
@@ -380,6 +397,109 @@ void having_two_pegouts_signs_only_one() throws Exception {
         );
     }
 
+    @Test
+    void onBestBlock_whenPegoutTxIsCached_shouldNotSignSamePegoutTxAgain() throws Exception {
+        Federation federation = TestUtils.createFederation(params, 9);
+        BtcTransaction pegoutBtcTx = TestUtils.createBtcTransaction(params, federation);
```

Comment:
i think there's no need to add `BtcTx` suffix here

Reply:
Sure, updated